### DISCUSSION TITLE
loading bar when downloading multiple models

### DISCUSSION THREAD

**Discussion by ouya-woelders** [open-webui/open-webui#5949] [General]
when i download multiple models a the same time, the loading bars start flashing. this is not a problem when only downloading one model at a time.

https://github.com/user-attachments/assets/775ac5d2-4f65-425e-a823-672faec1318b

heres a screen recording.

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/e4fdd9a6-65a5-4900-a075-03f38b7a0fbb)

Unable to reproduce, let us know your findings!

**Reply by ouya-woelders** [reply]
it happens when i reload it and navigate to the models tab when downloading multiple models.

**Reply by ouya-woelders** [reply]
it also seems to happen when i leave the tab for a while and come back to it. its running in docker on apple silicon. open-webui is opened in google chrome on a arm64 windows laptop

**Reply by ouya-woelders** [reply]
https://github.com/user-attachments/assets/1de7f72b-985d-4633-8cdc-7081f9291a03

it also seems to happen in the dropdown menu where you select models.
